apps/core/app/(default)/product/[slug]/_components/Breadcrumbs.tsx

```diff
@@ -1,4 +1,4 @@
-import { getProduct } from '~/client/queries/getProduct';
```

Comment:
@jorgemoya is it Ok that components like `Breadcrumbs`, `Gallery` & `Reviews` are still in PascalCase  comparing to  `related-products` as an example?

Reply:
Seemed to be a a bug with github, locally all files were kebab case. I forced a push to fix it.